src/pom.xml

```diff
@@ -135,7 +135,7 @@
         <spring.version>4.3.11.RELEASE</spring.version>
         <spring-security.version>4.2.3.RELEASE</spring-security.version>
         <!-- TODO: Raise log4j to version 2 -->
-        <log4j.version>1.2.17</log4j.version>
+        <log4j.version>2.10.0</log4j.version>
```

Comment:
Did you add an log4j config file for the tests as well?

Reply:
No, for the tests I just stuck with the `slf4j-simple` logger. But I converted the `log4j.properties` example in the archetype to a new `log4j2.xml`.